package.json

```diff
@@ -1,6 +1,7 @@
 {
-    "name": "@jupyterlite/xeus",
+    "name": "@jupyterlite/xeus-root",
```

Comment:
This change of name from `@jupyterlite/xeus` to `@jupyterlite/xeus-extension` should probably be documented somewhere, at least in the changelog.